question12/main.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const btn = document.getElementById('js-btn');
+
+async function callApi() {
+    try {
+        const res = await fetch('https://jsondata.okiba.me/v1/json/dFQLo210903231659');
+        const json = await res.json();
+        return json.data;
+    }
+    catch(error) {
+        wrapper.textContent = "データが取得できませんでした";
+        console.error(error);
+    }
+    finally{
+        btn.remove()
+    }
+}
```

Comment:
②I changed the inside of finally to remove the button.
（This is where I originally removed the loading image.）

Reply:
I think I'm good.👍